src/panel.scss

```diff
@@ -53,10 +53,6 @@ $block: #{$fd-namespace}-panel;
 
   &__button {
     font-size: 1rem;
-
-    @include fd-rtl() {
-      @include fd-icon('slim-arrow-left');
```

Comment:
You can keep the `fd-rtl` mixin and just rotate the icon, not add a new one.

Reply:
@InnaAtanasova thanks, fixed